pom.xml

```diff
@@ -16,9 +16,9 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/report-generic-chart-column-plugin</gitHubRepo>
         <chartjs.version>4.2.1.5</chartjs.version>
-        <jenkins.version>2.361.4</jenkins.version>
-        <!-- those twoo should be aligned -->
-        <bom.version>2.361.x</bom.version> <!-- Pick up common dependencies for the selected LTS line: https://github.com/jenkinsci/bom#usage -->
+        <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+        <jenkins.baseline>2.361</jenkins.baseline>
+        <jenkins.version>${jenkins.baseline}.4</jenkins.version>
```

Comment:
Can this be somehow connected also with `org.jenkins-ci.plugins:plugin` version?